charts/newrelic-infrastructure-v3/templates/kubelet/daemonset.yaml

```diff
@@ -161,6 +161,17 @@ spec:
             {{- with .Values.kubelet.extraEnv }}
             {{- toYaml . | nindent 12 }}
             {{- end }}
+            {{- /* Needed to populate clustername in integration metrics */}}
+            - name: "CLUSTER_NAME"
+              value: {{ include "newrelic.cluster" . | required "cluster name must be set" }}
+            - name: "NRIA_PASSTHROUGH_ENVIRONMENT"
+              value: "CLUSTER_NAME"
+            {{- /* Needed for autodiscovery since hostNetwork=false */}}
+            - name: "NRIA_HOST"
+              valueFrom:
+                fieldRef:
+                  apiVersion: "v1"
+                  fieldPath: "status.hostIP"
```

Comment:
Do we have control on the name for this env var? If so, i think we should use the same prefix we have in other places, like `NRI_KUBERNETES_NODE_IP`.

Reply:
Not really and I dislike it as well. The prefix is set [here](https://github.com/newrelic/nri-discovery-kubernetes/blob/f0e731a9bdd657c050d21f33b9ce97309eddb57c/internal/config/config.go#L25).

Changing it is easy but could lead to a breaking change :( 

Moreover I would chose something NRI_DISCOVERY

Reply:
The fact that it uses the same prefix as the agent worries me a bit... Are we sure the agent is not using the same variable for anything else?